.travis/build-deps.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/sh
+set -eux
+
+# Set defaults
+PREFIX="${PREFIX:-${HOME}/opt}"
+
+download_asio () {
+    if [ ! -d "download-cache/asio" ]; then
+        git clone https://github.com/chriskohlhoff/asio.git \
+            download-cache/asio
+    else
+        (
+            cd download-cache/asio
+	    if [ "$(git log -1 --format=%H)" != "${ASIO_VERSION}" ]; then
+                git checkout master
+                git pull
+                git checkout ${ASIO_VERSION}
+	    fi
+	)
+    fi
+}
+
+build_asio () {
+    (
+        if [ ! -L asio ]; then
+	    rm -Rf asio
+            ln -s download-cache/asio asio
+	fi
```

Comment:
Indenting looks off

Reply:
something was wrong with my vim style file. will fix it. Thanks!